core/coreunix/add_test.go

```diff
@@ -29,9 +23,9 @@ func TestAddRecursive(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
-	if k, err := AddR(node, path.Join(here, "test_data")); err != nil {
+	if k, err := AddR(node, "test_data"); err != nil {
```

Comment:
yeah, what changed was how the paths are consumed. I think this is erroneous, I'll fix it before we merge this PR.